### DISCUSSION TITLE
Display calculated field/column in form

### DISCUSSION THREAD

**Discussion by chrismb23** [Budibase/budibase#8400] [Help] [answered]
My data model (Budibase DB) contains a dynamic formula column that calculates the total price based on the previous columns (quantity and price). I'm using the auto generated new row screen to add data to my DB and want to display the calculated price on the same screen as soon as the user has entered the quantity and price. 

Is this possible at all, or should the new entry always be saved first?

**Comment by melohagan** [comment, marked as answer]
Hey @chrismb23 

> want to display the calculated price on the same screen as soon as the user has entered the quantity and price

This is for when a user enter values into a form, you want to display a sum? You could do that through the form field bindings.
Have a paragraph with this JS content: `return $("Form.Fields.MyQuantityField") * $("Form.Fields.MyPriceField");`

**Reply by NicolasFabre81** [reply]
Hello @melohagan , then how can one get the result value ?

**Reply by melohagan** [reply]
Hey @NicolasFabre81 I'm not sure what the context is here - what problem are you trying to solve?
